types/helpers/class-utils.d.ts

```diff
@@ -0,0 +1,3 @@
+export const getter: () => any;
+export const setter: (value: any) => void;
+export const applyProperties: (instance: any, params: any) => void;
```

Comment:
export default with the three of those ?

Reply:
https://github.com/wanadev/obsidian-core/pull/112/commits/1dc7fb53727793d5f9faeb2d7214675eef036a94

Reply:
I meant adding a new line at the end:

```
export const getter: () => any;
export const setter: (value: any) => void;
export const applyProperties: (instance: any, params: any) => void;

export default {
     getter,
     setter,
     applyProperties
}
```